server/database/queries/getPassword.js

```diff
@@ -0,0 +1,17 @@
+
+const dbconniction = require('../db_connection')
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'
+const getPassword = (email, callBack) => {
+  const sql = {
+    text: 'SELECT password FROM users where email=$1',
+    values: [email]
+  }
+  dbconniction.query(sql, (err, resulte) => {
+    if (err) {
+      return callBack(err)
+    }
+    return callBack(null, resulte.rows[0])
+  })
+}
```

Comment:
I suggest using promises instead of the callbacks, in which case it's gonna look like:
```js
return dbconniction.query(sql)
```
And that's it. Be careful about the `result.rows[0]` on line 13, you are not guaranteed that the user will be found, in which case postgres will return an empty result, not an error. See my comment in the controller on how to handle this.